addons/resource.language.en_gb/resources/strings.po

```diff
@@ -18319,3 +18319,18 @@ msgstr ""
 msgctxt "#38041"
 msgid "Mixer"
 msgstr "" 
+
+#: xbmc/video/dialogs/GUIDialogVideoBookmarks.cpp
+msgctxt "#38042"
+msgid "Rename bookmark"
+msgstr ""
+
+#: xbmc/dialogs/GUIDialogVideoBookmarks.cpp
+msgctxt "#38043"
+msgid "Removing all bookmarks"
+msgstr ""
+
+#: xbmc/dialogs/GUIDialogVideoBookmarks.cpp
+msgctxt "#38044"
+msgid "Removing bookmark"
```

Comment:
In that case Removing is better also less to translate and shorter translated string